app/assets/stylesheets/custom.scss

```diff
@@ -589,3 +589,9 @@ table {
     }
   }
 }
+
+.budget-heading {
```

Comment:
The `.budget-heading` selector is also used in `budgets/investments/_header`. Do we intend to add this image there as well?